common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
This CoA could use more space, why you didn't make it?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122810545-89b85b80-d2e0-11eb-81d2-0a01a71262fb.png)

</details>